docs/en/connector-v2/sink/Doris.md

```diff
@@ -138,5 +124,5 @@ sink {
 
 ### Next version
 
-- [Improve] Change Doris Config Prefix [3856](https://github.com/apache/incubator-seatunnel/pull/3856)
+- [Improve] Refactor some Doris Sink code as well as support 2pc and cdc [4235](https://github.com/apache/incubator-seatunnel/pull/4235)
```

Comment:
BTW, this is an incompatible change, so we better reflect it in the documentation as well.